### ISSUE TITLE
Integrate Welcome Bot

### ISSUE THREAD

**Issue by muthuannamalai12** [sahilrahman12/Terrorism-Analysis-with-Insights#2]
I can add a welcome bot config file having a proper message that will show up when any user will open up an issue or pull-request for the first time.
For reference, check out: https://github.com/apps/welcome

Please assign it to me as a part of Cross Woc

**Comment by vybhav72954** [comment]
Is this still open? @sahilrahman12 

Integrating this bot will take a few minutes

**Comment by muthuannamalai12** [comment]
> Is this still open? @sahilrahman12
> 
> Integrating this bot will take a few minutes
Yes, it takes a few minutes @vybhav72954 . But I need a code of conduct for my custom message shown by welcome bot. For that, I have opened a PR no: #4. Once it is merged I will complete it within few minutes.
